guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -106,6 +106,25 @@ stage:
       _merge: true
 ```
 
+### `ENABLE_GOOGLE_ANALYTICS`
+
+-  **Default**—_false_
+-  **Version**—Magento 2.1.4 and later
+
+```yaml
```

Comment:
This example was not introduced like the others. Need to move this example after the definitions.